ios/Button.swift

```diff
@@ -10,6 +10,7 @@ class Button : RCTView {
   @objc lazy var onCancel: RCTBubblingEventBlock = { _ in }
   @objc lazy var onPressStart: RCTBubblingEventBlock = { _ in }
   @objc lazy var onLongPress: RCTBubblingEventBlock = { _ in };
+  @objc lazy var onLongPressEnded: RCTBubblingEventBlock = { _ in };
```

Comment:
sorry I  got so late to this review
my issue here is that I need to do an operation from js in between the button is pressed and the press ended, so I need a way to listen to the onLongPressHandler gesture `ended` to stop the operation, that's why I'm using a callback. Passing a boolean is not enough because that wouldn't give me that ability.

Reply:
yes, @estebanmino that is correct, one prop is not sufficient so I expect to have 2:
`shouldLongPressEndPress=false` + `onLongPressEnded={() => { /* do sth */ }}` (as you have now). 

I just suggest that adding a callback should not change the behavior of the component. 

In other words, I want to split what you do into two parts: 1. changing the behavior (that passing a threshold should not end gesture) 2. adding a callback. 

Currently, those two elements are connected in one prop what is misleading. E.g. when I will want to disable this behavior (that passing a threshold should not end gesture) here (or anywhere), removing callback will be the last thing I can think of.

Reply:
perfect!